File: giorno2/es1.js
// Giorno 2 - Es 1
let a = parseInt(prompt("Inserisci A:"));
let b = parseInt(prompt("Inserisci B:"));
let c = parseInt(prompt("Inserisci C:"));

if (a > b && a > c) {
    console.log("Il maggiore è:", a);
} else if (b > a && b > c) {
    console.log("Il maggiore è:", b);
} else {
    console.log("Il maggiore è:", c);
}

File: giorno2/es1.py
a=int(input("Inserisci A: "))
b=int(input("inserisci B: "))
c=int(input("Inserisci C: "))
if a > b and a>c:
    print("Il maggiore è: ", a)
elif b>a and b>c:
    print("il maggiore è", b)
else:
    print("il maggiore è:", c)